batch/batch/client.py

```diff
@@ -77,33 +77,49 @@ def id(self):
     def attributes(self):
         return self._async_batch.attributes
 
+    def cancel(self):
+        async_to_blocking(self._async_batch.cancel())
+
+    def status(self):
+        return async_to_blocking(self._async_batch.status())
+
+    def wait(self):
+        return async_to_blocking(self._async_batch.wait())
+
+    def delete(self):
+        async_to_blocking(self._async_batch.delete())
+
+
+class BatchBuilder:
+    @classmethod
+    def from_async_builder(cls, builder):
+        b = object.__new__(cls)
+        b._async_builder = builder
+        return b
+
+    def __init__(self, client, attributes, callback):
+        self._async_builder = aioclient.BatchBuilder(client, attributes, callback)
+
     def create_job(self, image, command=None, args=None, env=None, ports=None,
                    resources=None, tolerations=None, volumes=None, security_context=None,
                    service_account_name=None, attributes=None, callback=None, parents=None,
                    input_files=None, output_files=None, always_run=False, pvc_size=None):
-        coroutine = self._async_batch.create_job(
+        if parents:
+            parents = [parent._async_job for parent in parents]
+
+        async_job = self._async_builder.create_job(
```

Comment:
Could this be simplified using **kwargs (since argument list besides self is the same)

Reply:
I don't know what the preference of the team is for this. It has always been explicit in previous iterations of the client, api code

Reply:
K